src/game/engine/target.h

```diff
@@ -70,6 +70,7 @@ class TargetClass
 
 inline target_t Make_Target(RTTIType type, int id)
 {
+    // return ((type & 0xFF) << 24) | (id & 0xFFFFFF);
```

Comment:
This comment is redundant as its the same as the code below written another way. Either change the code or don't bother with the comment, not both.